src/app/wines/[wineId]/components/ReviewCard.tsx

```diff
@@ -10,7 +10,9 @@ import InputRange from '@/components/InputRange';
 import ProfileImg from '@/components/ProfileImg';
 import { formatToTimeAgo } from '@/libs/formmatToTimeago';
 
-export default function ReviewCard() {
+import { ReviewType } from '../types';
+
+export default function ReviewCard({ review }: { review: ReviewType }) {
```

Comment:
이렇게 review를 받아서 바로 사용해도 좋지만, 구조분해할당으로 미리 나눠서 사용하기도 하더라구요!
이미 알고 계시겠지만 코드를 확인하다 한 번 생각나서 말씀드려봅니다~

```suggestion
  const { user: { nickname }, createdAt, ... } = review;
```

Reply:
오, 구조분해할당 적용시켜 놓는 게 더 깔끔할 것 같군요!